src/Microsoft.Health.Dicom.Blob/Features/ExternalStore/ExternalBlobClient.cs

```diff
@@ -98,6 +97,6 @@ public string GetServiceStorePath(string partitionName)
     {
         return _isPartitionEnabled ?
             _externalStoreOptions.StorageDirectory + partitionName + "/" :
-            _externalStoreOptions.StorageDirectory + PartitionEntry.Default.PartitionName + "/";
+            _externalStoreOptions.StorageDirectory + "/";
```

Comment:
Does `StorageDirectory` already end with a `/`? Should we do some massaging of this value?